gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -443,6 +454,11 @@ properties:
       - "_terms.yaml#/cytoband/common"
     type: string
 
+  days_to_test:
+    $ref:
+      - "_terms.yaml#/days_to_test/common"
+    type: integer
+
```

Comment:
seems like we can do some validation here for min/max, e.g. e.g. -32872 and 32872

Reply:
updated